redskyctl/internal/commands/run/view.go

```diff
@@ -129,26 +138,33 @@ https://docs.stormforger.com/guides/getting-started/`,
 	}
 
 	o.generatorModel.ContainerResourcesSelectorInput = out.FormField{
-		Prompt:       "Specify labels to control discovery of memory and CPU parameters:",
-		Placeholder:  "( e.g. component=api )",
-		Instructions: []string{"Leave blank to select all resources", "pgup: back", "enter: continue"},
+		Prompt:      "Specify labels to control discovery of memory and CPU parameters:",
+		Placeholder: "( e.g. component=api )",
+		Instructions: []interface{}{
```

Comment:
Seems like the whitespace alignment is off/changed? Just to double check gofmt was run on this right?

Reply:
The array changed from a single line to a multi-line declaration so the alignment reset.